torchci/lib/bot/mergeBot.ts

```diff
@@ -46,19 +47,28 @@ function mergeBot(app: Probot): void {
       await dispatchEvent("try-revert");
       await reactOnComment(ctx, "+1");
     }
+    if (commentBody.match(rebaseCmdPat) && ctx.payload.comment.user.login == "clee2000") {
+      if (!ctx.payload.issue.pull_request) {
+        // Issue, not pull request.
+        await reactOnComment(ctx, "confused");
+        return;
+      }
+      await dispatchEvent("try-rebase");
+      await reactOnComment(ctx, "+1");
+    }
   });
   app.on(["pull_request_review.submitted", "pull_request_review.edited"], async (ctx) => {
     const reviewBody = ctx.payload.review.body;
```

Comment:
Do we need this webhook or would the issue_comment.created be enough? my @malfet would know

Reply:
I think it's fine to use issue_comment.created here as a probot trigger and then only if it matches the right stuff does it trigger the GHA.